website/content/docs/platform/k8s/vso/secret-transformation.mdx

```diff
@@ -0,0 +1,234 @@
+---
+layout: docs
+page_title: Vault Secrets Operator Secret Transformation
+description: >-
+  Secret data transformation.
+---
+
+# Secret data transformation
+
+Utilizing advanced templating and data filters, the Vault Secrets Operator for Kubernetes (VSO) can
+transform source secret data, secret metadata, resource labels and annotations into a format that is
+compatible with your application. All secret data sources are supported. Secret transformations can
+be specified directly within a secret custom resource (CR), or by references to one or more
+`SecretTransformation` custom resource instances, or both.
+
+## Templating
+
+VSO utilizes the data-driven [templates for Golang](https://pkg.go.dev/text/template) to generate
+secret data output. The template data input holds the secret data, secret metadata, resource labels
+and annotations.
+
+VSO provides access to a large library of template functions, some of which are documented
+[below](#template-functions).
+
+### Secret data input
+
+Secret data is accessed through the `.Secrets` input member. It contains a map of secret key to
+its secret value. The data contains confidential information.
```

Comment:
Just thinking out loud about the last two sentences:

```suggestion
Secret data is accessed through the `.Secrets` input member. It contains a map of secret 
key-value pairs, which are assumed to be sensitive information fetched from a [secret source](/vault/docs/platform/k8s/vso/sources).
```

Reply:
Fixed in fc2d665d6fcae44c531c415e182b9a5b5318701b